docs/understanding-tracking-design/create-tracking-plan-with-tracking-scenarios/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Creating a Tracking Plan with Tracking Scenarios"
+sidebar_label: "🆕 Creating a Tracking Plan"
```

Comment:
Could you use #509 instead once that is merged?

Reply:
(Do keep a short `sidebar_label`!)

Reply:
Done